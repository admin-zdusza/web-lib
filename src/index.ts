export { Brand } from './components/brand/Brand';
export { TimeoutError } from './components/timeout-error/TimeoutError'
export { timeoutOccured } from './components/timeout-error/actions-creators';
export { timeoutReducer } from './components/timeout-error/reducers';
