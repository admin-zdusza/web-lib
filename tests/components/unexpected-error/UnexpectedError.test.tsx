import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import { UnexpectedError } from "../../../src";

test("should render timeout error closable", () => {
  const component = <UnexpectedError closable={true} dispatch={jest.fn()} />;
  const { asFragment } = render(component);
  expect(asFragment()).toMatchSnapshot();
});

test("should render timeout error non closable", () => {
  const component = <UnexpectedError closable={false} dispatch={jest.fn()} />;
  const { asFragment } = render(component);
  expect(asFragment()).toMatchSnapshot();
});
