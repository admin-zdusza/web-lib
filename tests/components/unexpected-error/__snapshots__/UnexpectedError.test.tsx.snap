// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render timeout error closable 1`] = `<DocumentFragment />`;

exports[`should render timeout error non closable 1`] = `<DocumentFragment />`;
